runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -92,6 +99,35 @@ class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 	virtual bool preMonitorTableSlotRead(J9JavaVM *vm, j9object_t *srcAddress);	
 #endif	
 
+	/* New methods */
+	bool preObjectStoreImpl(J9VMThread *vmThread, J9Object *destObject, fj9object_t *destAddress, J9Object *value, bool isVolatile);
+	bool preObjectStoreImpl(J9VMThread *vmThread, J9Object **destAddress, J9Object *value, bool isVolatile);
+
+	/* New methods */
+	void rememberObjectToRescan(MM_EnvironmentBase *env, J9Object *object);
+
+	MMINLINE bool isSATBBarrierActive(MM_EnvironmentBase* env)
+	{
+		return !_extensions->sATBBarrierRememberedSet->isGlobalFragmentIndexPreserved(env);
+	}
+
+	MMINLINE bool isIncrementalUpdateBarrierActive(MM_EnvironmentBase* env)
+	{
+		return (_extensions->concurrentMark &&
+				(vmThread->privateFlags & J9_PRIVATE_FLAGS_CONCURRENT_MARK_ACTIVE) &&
```

Comment:
we can remove _extensions->concurrentMark now. it's being replaced by more specific configuration->isIncrementalUpdateBarrierEnabled check

Reply:
removed concurrentMark check.